src/core/InstrumentFunctions.cpp

```diff
@@ -349,6 +349,9 @@ void InstrumentFunctionArpeggio::processNote( NotePlayHandle * _n )
 		return;
 	}
 
+	// make sure note is handled as arp-base-note, even
+	// if we didn't add a sub-note so far
```

Comment:
I think it'd be better to copy the comment from [Line 404](https://github.com/LMMS/lmms/pull/4487/files#diff-433d51d20830d46278dac98663089e30L404).

Reply:
OK. Fixed...

Reply:
No, I copied the wrong line. :O